assets/css/anchor-link.css

```diff
@@ -1,45 +1,47 @@
 .anchored-link {
-    display: none;
-    background-image: url(/assets/icons/link.svg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: 100% 100%;
-    text-decoration: none;
-    margin-left: 10px;
-    vertical-align: middle;
-    width: 0.8em;
-    height: 0.8em;
-    max-width: 24px;
-    max-height: 24px;
-}
-.anchored-link:hover {
+  display: none;
+  background-image: url(/assets/icons/link.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+  text-decoration: none;
+  margin-left: 10px;
+  vertical-align: middle;
+  width: 0.8em;
+  height: 0.8em;
+  max-width: 24px;
+  max-height: 24px;
+
+  & .anchored-link:hover {
```

Comment:
CSS nesting is still not supported by all browsers. Do we have a preprocessor in place until that happens?

Reply:
It's supported by all major browsers, what other browsers should I look for?

Reply:
17% of browsers are still not supporting it, we can't really make the jump yet: https://caniuse.com/css-nesting

Reply:
Yeah somehow I thought it has 93% + 
I've added postcss-preset-env with postcss to generate css compatible with all browsers :D

Reply:
Unfortunately that requires for us to use npm, and one of the reasons why we kept everything in jekyll/scss was to not add more dependencies. So if Hugo doesn't have anything native I thing you should revert the css nesting and leave it as is.

Reply:
I've reverted it :(
I really like using it, it's so much cleaner

Btw the archive generator and mirrorlist generator use javascript and currently it has 1 dependency to parse yaml, is that an issue? If so we could use json instead